o2a/converter/oozie_converter.py

```diff
@@ -89,7 +89,10 @@ def convert(self, as_subworkflow=False):
         self.property_parser.parse_property()
         self.parser.parse_workflow()
 
-        self.apply_transformers()
+        logging.info(f"Applying transformers")
```

Comment:
I thought it would be clearer if they were separate, because now the method would have to have a very long name. You can see the use case in the next commit.

Reply:
> the method would have to have a very long name
The need for extracting some code to a method should be a separate concern from naming the method.
Feeling that you can only come up with a long name for the method should not discourage you from extracting.

Reply:
Can you provide name suggestions for these two methods?

Reply:
How about:
```
apply_preconvert_transformers()
apply_postconvert_transformers()
```
?

Reply:
They do not describe specifically what the function does, but it seems to me that these are good enough names.

Reply:
Thanks. I will update this PR.